### DISCUSSION TITLE
hub-mirror.c.163.com

### DISCUSSION THREAD

**Discussion by gzm55** [docker-practice/docker-registry-cn-mirror-test#12] [General]
测试镜像从hub-mirror.c.163.com拉不下来, 应该是某个 blob 不完整, 换另外 tag 就好了

[a comment by an automated account is not shown]

**Comment by khs1994** [comment]
我不清楚 Docker Hub 免费（匿名）账户限流会不会对镜像服务产生影响。

* https://www.docker.com/pricing
